src/UI/components/map/mapView/layerUtils.tsx

```diff
@@ -172,7 +172,7 @@ export const updateOverlayLayers = (
 
   const allLayers = map?.getAllLayers();
   newLayers.forEach((l) => {
-    map?.getLayers().insertAt(allLayers ? allLayers.length - 1 : 0, l);
+    map?.getLayers().insertAt(allLayers ? allLayers.length - 2 : 0, l);
```

Comment:
Hmm, these changes make me a bit uncomfortable... I assume these numbers need incrementing when we add a layer? Is that documented somewhere?

Reply:
It only increments if we add any more permanent layers (e.g. here it was the area select layer) - a developer will notice because their new layer or the point layer will appear below overlays. It's not documented anywhere but I'm not sure it will help because it will depend on where the layer needs to go in the stack - here I'm just saying that overlays need to be inserted below the point and area select layers.

Reply:
Ah, makes sense. Thanks.